app/views/users/show.html.erb

```diff
@@ -1,2 +1,31 @@
-<h1> Here is an author with its corresponding bio and his/her posts
-</h1>
+<div class="user-posts-wrapper">
+  <h1 class="header-text"> Here is an author with her/his corresponding bio and his/her posts </h1>
+  <div class="user-posts">
+    <div class="author-info user-link-container">
+        <img class="post-box" src ="/box-post.png" alt="<%= @user.name %>'s profile picture">
+        <div class="user-link-wrapper">
+          <p class="username-link"> <%= link_to @user.name, user_path(@user) %> </p>
+          <p class="posts-counter">  Number of posts: <%= @user.posts_counter %> </p>
+        </div>
+    </div>
+
+    <div class="bio-container">
+      <p class="bio-title"> Bio </p>
+      <p class="bio-text"> <%= @user.bio%> </p>
+    </div>
+
+      <% @user.three_recent_posts.each do |post| %>
+        <div class="post-container">
+          <p class="post-title"> <%= link_to post.title, user_post_path(@user, post) %> </p>
+          <p class="post-text"> <%= post.text %> </p>
+          <p class="comments-likes-count">
+            Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
+          </p>
+        </div>
+      <% end %>
+
+  <button class="btn btn-all-posts">
+    <%= link_to 'See all posts', user_posts_path(@user) %>
+  </button>
```

Comment:
- [ ] Please align the elements position according to the wireframe. You can use flexbox. Right now, your project looks like: 
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/88402fd1-450e-4610-977e-a29f0b4930a4)
the wireframe:
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/03fc84a6-7d3b-4f42-b251-ec75fd028b67)

Reply:
css added